test/verify/check-machines

```diff
@@ -41,6 +43,23 @@ def getNetworkDevice(m):
 
     return device
 
+def readFile(name):
+    content = ''
+    if os.path.exists(name):
+        with open(name, 'r') as f:
+            content = f.read().replace('\n', '')
+    return content
+
+def createUser(machine, user_group):
+    user_name = 'test_' + user_group + '_' + randomString()
+    machine.execute(
+        'sudo useradd -G {} {} && echo "{}" | sudo passwd --stdin {}'.format(user_group, user_name, "foobar", user_name))
```

Comment:
Please use `{0}` and `{1}` so that you don't have to repeat the user name three times. Also, don't run these through sudo, `m.execute()` is already root.

Reply:
Done.
Also changed the passwd command because the --stdin parameter did not exist on debian and ubuntu images.